src/sentry/tasks/weekly_reports.py

```diff
@@ -271,11 +271,11 @@ def zerofill_data(data):
     request = Request(dataset=Dataset.Outcomes.value, app_id="reports", query=query)
     data = raw_snql_query(request, referrer="weekly_reports.outcomes")["data"]
 
-    # TODO(isabella): remove debug logging for sentry
-    if ctx.organization.slug == "sentry":
+    if features.has("organizations:weekly-report-logs", ctx.organization):  # TODO(isabella): remove
```

Comment:
@isabellaenriquez why are we feature flagging the logging at all? Are we worried it might break or something like the recent incident?

Reply:
@scefali it's more just because i only want to log for a specific org + sentry, rather than having it log for every single user that gets weekly reports. ideally i would test this today and then take the logging out so that it doesn't get hit on monday when the regularly scheduled weekly reports go out, but not sure if i'll be able to merge + test + make the pr to take them out and merge it in all before the end of today